portfolio/src/main/webapp/stylenew.css

```diff
@@ -0,0 +1,359 @@
+* {box-sizing:border-box}
```

Comment:
Why did you name this file `stylenew.css` and not update `style.css` instead?

Reply:
I used to also have an old version of my index page in the project, and I kept it for that, will change it in PR 7 to syle.css though